gensim/test/test_ldamodel.py

```diff
@@ -253,6 +255,12 @@ def testShowTopics(self):
                 self.assertTrue(isinstance(k, six.string_types))
                 self.assertTrue(isinstance(v, float))
 
+        #testing print_topic
```

Comment:
create new test test_print_topic in new class TestBaseTopicModel.
TestLdaModel, testLSI and testHDP should inherit from TestBaseTopicModel.